code/__HELPERS/traits.dm

```diff
@@ -62,6 +62,7 @@ Remember to update _globalvars/traits.dm if you're adding/removing/renaming trai
 
 //mob traits
 #define TRAIT_PACIFISM			"pacifism"
+#define TRAIT_WATERBREATH       "waterbreathing"
```

Comment:
It would make more sense to add it here: https://github.com/ParadiseSS13/Paradise/blob/99a53306847df59f054931bbd07c045e32f2f58a/code/__DEFINES/genetics.dm#L144

Reply:
It was suggested to me to use this version as the framework for traits was recently added and it also lays the ground work for future use in items and other things.